tests/aws/services/stepfunctions/templates/scenarios/statemachines/retry_interval_features_max_attempts_zero.json5

```diff
@@ -0,0 +1,30 @@
+{
+  "Comment": "RETRY_INTERVAL_FEATURES_MAX_ATTEMPTS_ZERO",
+  "StartAt": "LambdaTask",
+  "States": {
+    "LambdaTask": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "Parameters": {
+        "FunctionName.$": "$.FunctionName",
```

Comment:
clarification question: Is the `.$` an indicator for template variable replacement?

Reply:
Correct, it's a StepFunctions' language feature which binds to the identifier that precedes to the argument assigned ("FunctionName": "lambda_function_name").